Scripts~/windows/.export-server-sdk.ps1

```diff
@@ -5,42 +5,43 @@ param(
 	[string] $ServerSdkVersion
 )
 
-$ROOT_DIR='.'
-$SERVER_SDK_FILENAME="GameLift-CSharp-ServerSDK-UnityPlugin-$ServerSdkVersion.zip";
-$DESTINATION_PATH="$ROOT_DIR\$SERVER_SDK_FILENAME"
-$TEMP_PATH="C:\Temp"
-$TEMP_ZIP_PATH = "$TEMP_PATH\GameLiftServerSDK.zip"
-$TEMP_EXTRACTED_PATH = "$TEMP_PATH\GameLiftServerSDK"
-$S3_SERVER_SDK_BUCKET="https://gamelift-server-sdk-release.s3.us-west-2.amazonaws.com/unity"
-$S3_SERVER_SDK_DOWNLOAD_LINK="$S3_SERVER_SDK_BUCKET/$SERVER_SDK_FILENAME"
-
-& "$PSScriptRoot\.verify-working-directory.ps1"
-if ($LASTEXITCODE -ne 0) { exit $LASTEXITCODE }
+$ROOT_DIR=Resolve-Path "$PSScriptRoot\..\.."
+$BUILD_DIR="$ROOT_DIR\build"
+$SERVER_SDK_FILENAME="GameLift-CSharp-ServerSDK-UnityPlugin-$ServerSdkVersion";
+$DESTINATION_PATH="$BUILD_DIR\$SERVER_SDK_FILENAME.zip"
+$TEMP_PATH="C:\Temp\AmazonGameLiftPluginUnity"
+$TEMP_ZIP_PATH = "$TEMP_PATH\$SERVER_SDK_FILENAME.zip"
+$TEMP_EXTRACTED_PATH = "$TEMP_PATH\$SERVER_SDK_FILENAME"
+$SERVER_SDK_S3_LINK_PREFIX="https://gamelift-server-sdk-release.s3.us-west-2.amazonaws.com/unity"
+$SERVER_SDK_S3_DOWNLOAD_LINK="$SERVER_SDK_S3_LINK_PREFIX/$SERVER_SDK_FILENAME.zip"
 
 if (Test-Path -Path $DESTINATION_PATH)
 {
 	Write-Host "Amazon GameLift Server SDK zip has already been prepared, skipping download and extraction." -ForegroundColor DarkYellow
 	exit 0
 }
 
-echo "Preparing Amazon GameLift Server SDK for release..."
+Write-Host "Preparing Amazon GameLift Server SDK for release..."
 
 if (-Not (Test-Path -Path $TEMP_ZIP_PATH) )
 {
-	echo "Downloading Amazon GameLift Server SDK $ServerSdkVersion..."
-	iwr $S3_SERVER_SDK_DOWNLOAD_LINK -OutFile $TEMP_ZIP_PATH
+	New-Item -Force -Type Directory -Path $TEMP_PATH -ErrorAction Stop
+	Write-Host "Downloading Amazon GameLift Server SDK $ServerSdkVersion..."
+	# Download link should be public and require no credentials
+	iwr $SERVER_SDK_S3_DOWNLOAD_LINK -OutFile $TEMP_ZIP_PATH
 }
 else
 {
-	echo "Amazon GameLift Server SDK is already downloaded and stored at $TEMP_ZIP_PATH, skipping download."
+	Write-Host "Amazon GameLift Server SDK is already downloaded and stored at $TEMP_ZIP_PATH, skipping download."
 }
 
-echo "Extracting and removing license file from Server SDK zip..."
+Write-Host "Extracting and removing license file from Server SDK zip..."
 
-Expand-Archive -LiteralPath $TEMP_ZIP_PATH -DestinationPath $TEMP_EXTRACTED_PATH -ErrorAction Stop
-Remove-Item -LiteralPath "$TEMP_EXTRACTED_PATH\LICENSE.txt" -Force -ErrorAction Stop
+Expand-Archive -Force -LiteralPath $TEMP_ZIP_PATH -DestinationPath $TEMP_EXTRACTED_PATH -ErrorAction Stop
+# See internal runbook for details
```

Comment:
This is going to be publicly released, right? I don't think we can have this comment.

Reply:
```suggestion
```